src/net/sourceforge/kolmafia/persistence/ConsumablesDatabase.java

```diff
@@ -384,6 +390,16 @@ else if (consumable.getConsumptionType() == ConsumptionType.DRINK) {
       }
     }
 
+    if (KoLCharacter.inElevenThingIHateAboutU()) {
+      if (consumable.getConsumptionType() == ConsumptionType.EAT) {
+        start =
+            end = (int) Math.round(1.5 * KoLCharacter.getEyeosity(name) * consumable.getFullness());
```

Comment:
You said that Wiki and Discord agree that this is ceil().
Given that, I respectfully request that you use Math.ceil, not Math.round.